app/forms/support_interface/conditions_form.rb

```diff
@@ -0,0 +1,78 @@
+module SupportInterface
+  class ConditionsForm
+    class OfferConditionField
+      include ActiveModel::Model
+
+      attr_accessor :id, :text
+
+      validates :text, length: { maximum: 255, too_long: ->(c, _) { "Condition #{c.id + 1} must be %{count} characters or fewer" } }
+    end
+
+    include ActiveModel::Model
+
+    attr_accessor :application_choice, :standard_conditions, :further_conditions, :audit_comment
+
+    NUMBER_OF_FURTHER_CONDITIONS = 4
+
+    validates :application_choice, presence: true
+    validates :audit_comment, presence: true
+
+    def self.build_from_application_choice(application_choice, attrs = {})
+      attrs = {
+        application_choice: application_choice,
+        standard_conditions: standard_conditions_from(application_choice.offer),
+        further_conditions: further_conditions_from(application_choice.offer),
+      }.merge(attrs)
+
+      new(attrs).tap(&:setup_further_conditions)
```

Comment:
I don't get why `setup_further_conditions` is invoked here given we have assigned further conditions on line 24 🤔

Reply:
It's there to fill out the data structure that drives the list of conditions on the form. It's not well named. I gonna rename it `add_slots_for_new_conditions` to try and make it clearer for now. However I think this needs to be revisited given that we want to up the limit to 20 conditions (and rejig the UI in the process).

Reply:
Ok that makes sense now 👍